tools/update-libuv.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
Does it make any sense to group al of the scripts to do dep updates in a single directory? Maybe `tools/dep-updaters`.

That would also give us a place to have a README.md that provides some info on what automatic updates are in place, and related info. For example in this PR there is some discussion about how to handle situation X and it would be good to have that documented in a discoverable place.

Reply:
I think it does! I can open a PR moving all scripts once the pending PRs are merged

Reply:
What got me started was I was thinking some additional doc/description for this PR would make sense. Could you move this one there, add that doc and then do the move for the rest in the follow on PR?

Reply:
Done, moved to a new folder and added a [README](https://github.com/nodejs/node/blob/1d13597758cce9c00bbf5e1062e67eeed3017638/tools/dep_updaters/README.md) with documentation. cc @lpinca could you check if the instructions are OK?